src/styles/main.scss

```diff
@@ -1,3 +1,74 @@
+$row-width: 100px;
+$gap: 1px;
+$days: (
+  "mon": 0,
+  "tue": 1,
+  "wed": 2,
+  "thu": 3,
+  "fri": 4,
+  "sat": 5,
+  "sun": 6,
+);
+
+@function calendarWidth($boxWidth, $gap) {
+  @return (($boxWidth * 7) + ($gap * 6));
+}
+
+@mixin boxSize($value) {
+  height: #{$value};
+  width: #{$value};
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
```

Comment:
If you add to this class `margin: 0 auto;` then the calendar is aligned horizontally in the center.